kubernetes-deployment.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes Deployment
+
+### Prerequisites
+Docker Community Edition: Tested on docker version ```18.09.6```
```

Comment:
IMO, Prerequisites is Docker daemon with docker client. We have tested with Docker community edition (18.09.6)

Reply:
Replaced with the same.